{"additions": 86, "auther_ref": "P8RndmEnginePtr", "auther_sha": "450e30c9ef255be8268ac36ed47fbb13d6c990b8", "author": "mseidel42", "body": "#### PR description:\r\n\r\nPythia 8.310 expects a shared_ptr for the random number generator now, see corresponding https://github.com/cms-sw/cmsdist/pull/8938\r\n\r\nI changed `unique_ptr` with `shared_ptr` in various places and it compiles but \r\n`runTheMatrix.py -j 10 -l 14234.0`\r\ngives segfault in\r\n`#5  0x00007f6a1e22000a in gen::P8RndmEngine::setRandomEngine (v=0x7f6a20b25b80, this=0x0) at /build/mseidel/CMSSW_14_0_X_2024-01-22-2300/src/GeneratorInterface/Pythia8Interface/interface/P8RndmEngine.h:34`\r\n\r\n@wddgit I think this touches a lot of your work, could you have a look, please?\r\n", "branch": "master", "changed_files": 26, "comments": 3, "commits": 1, "created_at": "1706189590", "deletions": 83, "labels": ["core-pending", "fastsim-pending", "generators-pending", "pending-signatures", "tests-pending", "orp-pending", "pdmv-pending", "upgrade-pending", "code-checks-rejected"], "milestone": "CMSSW_14_0_X", "number": 43786, "release-notes": [], "review_comments": 0, "state": "open", "title": "Adaptions for Pythia 8.310", "updated_at": "1706198170", "user": "mseidel42"}